packages/orbit-components/src/ListChoice/__tests__/index.test.js

```diff
@@ -1,119 +1,78 @@
+    const description = "Further description";
+    render(<ListChoice title={title} description={description} />);
+    expect(screen.getByText(description)).toBeInTheDocument();
   });
+
   it("should render checkbox with checked false", () => {
-    expect(component.find("Checkbox").prop("checked")).toBe(true);
+    render(<ListChoice title={title} selectable />);
+    expect(screen.getByRole("checkbox", { checked: false })).toBeInTheDocument();
   });
+
   it("should NOT execute onClick method", () => {
-    component.simulate("click");
+    render(<ListChoice title={title} disabled onClick={onClick} dataTest={dataTest} />);
+    const component = screen.getByTestId(dataTest);
+    userEvent.click(component);
     expect(onClick).not.toHaveBeenCalled();
   });
+
   it("should execute onClick method", () => {
-    component.setProps({ disabled: false });
-    component.simulate("click");
+    render(<ListChoice title={title} onClick={onClick} dataTest={dataTest} />);
+    const el = screen.getByTestId(dataTest);
+    userEvent.click(el);
     expect(onClick).toHaveBeenCalled();
   });
-  it("should have role button when not selectable", () => {
-    render(
-      <ListChoice
-        title={title}
-        description={description}
-        selectable={false}
-        icon={<Accommodation />}
-        onClick={onClick}
-        dataTest={dataTest}
-      />,
-      screen,
-    );
 
-    const listChoice = screen.querySelector(`[data-test="${dataTest}"]`);
-    expect(listChoice?.getAttribute("role")).toBe("button");
+  it("should have role button when not selectable", () => {
+    render(<ListChoice title={title} onClick={onClick} />);
+    expect(screen.getByRole("button")).toBeInTheDocument();
   });
 
   it("should have role checkbox when selectable", () => {
-    render(
-      <ListChoice
-        title={title}
-        description={description}
-        selectable
-        selected={false}
-        icon={<Accommodation />}
-        onClick={onClick}
-        dataTest={dataTest}
-      />,
-      screen,
-    );
-
-    const listChoice = screen.querySelector(`[data-test="${dataTest}"]`);
-    expect(listChoice?.getAttribute("role")).toBe("checkbox");
-    expect(listChoice?.getAttribute("aria-checked")).toBe("false");
+    render(<ListChoice title={title} selectable dataTest={dataTest} />);
+    expect(screen.getByTestId(dataTest)).toHaveAttribute("role", "checkbox");
   });
 
-  it("should have aria-checked true when selectable and selected", () => {
-    render(
-      <ListChoice
-        title={title}
-        description={description}
-        selectable
-        selected
-        icon={<Accommodation />}
-        onClick={onClick}
-        dataTest={dataTest}
-      />,
-      screen,
-    );
+  it("should have focus", () => {
+    render(<ListChoice title={title} selectable dataTest={dataTest} />);
+    userEvent.tab();
+    expect(screen.getByTestId(dataTest)).toHaveAttribute("tabindex", "0");
```

Comment:
The `tabindex` check is unnecessary because it's an implementation detail which might change, you're only interested in whether the element has focus, like the test says.